CHANGES.md

```diff
@@ -1,6 +1,12 @@
 Change Log
 ==========
 
+### 1.47 - 2018-07-02
+
+##### Additions :tada:
+* `PostProcessStage` has a `selectedFeatures` property which is an array of primitives used for selectively applying a post-process stage. In the fragment shader, use the function `bool czm_selected(vec2 textureCoordinates` to determine whether or not the stage should be applied at that fragment.
```

Comment:
> In the fragment shader, use the function `bool czm_selected(vec2 textureCoordinates` to determine whether or not the stage should be applied at that fragment.

This is too much detail for CHANGES.md.  Just have this documented where needed.  Maybe a Sandcastle example with a custom stage as well.